File: auth-service/src/main/java/com/eirsteir/coffeewithme/authservice/service/AccountService.java
package com.eirsteir.coffeewithme.authservice.service;

import com.eirsteir.coffeewithme.authservice.domain.Account;
import com.eirsteir.coffeewithme.authservice.web.request.UserRegistrationRequest;
import org.springframework.stereotype.Service;

@Service
public interface AccountService {

    Account registerAccount(UserRegistrationRequest registrationRequest);

}

File: auth-service/src/main/kotlin/com/eirsteir/coffeewithme/authservice/service/AccountService.kt
package com.eirsteir.coffeewithme.authservice.service

import com.eirsteir.coffeewithme.authservice.domain.Account
import com.eirsteir.coffeewithme.authservice.web.request.RegistrationRequest
import org.springframework.stereotype.Service

interface AccountService {
    fun registerAccount(registrationRequest: RegistrationRequest): Account
}